torch/functional.py

```diff
@@ -374,18 +374,9 @@ def stft(input: Tensor, n_fft: int, hop_length: Optional[int] = None,
          win_length: Optional[int] = None, window: Optional[Tensor] = None,
          center: bool = True, pad_mode: str = 'reflect', normalized: bool = False,
          onesided: Optional[bool] = None,
-         return_complex: Optional[bool] = None) -> Tensor:
+         return_complex: bool = True) -> Tensor:
```

Comment:
The default needs to be `False` here, otherwise this is a silent BC breaking change.

Reply:
Or actually, no it still needs to be `None` so complex tensors pass through correctly.

Reply:
I don't understand. The intent of this PR is to only return complex Tensors from `stft`. So what will having `return_complex: Optional[bool] = None` accomplish here, since it cannot be `False` later on anyway?

Reply:
There is also the case of complex input which already reaturns complex output, so isn't being change. There's no reason to break that usage.